<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QWidget" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manga Manager</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="left_section" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#2a2d31;
color:#faf9f9;</string>
     </property>
     <layout class="QVBoxLayout" name="left_section_layout">
      <item>
       <widget class="QLabel" name="logo_label">
        <property name="font">
         <font>
          <family>Impact</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Manga Manager</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="libraries_list_label">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">margin-top:10px;</string>
        </property>
        <property name="text">
         <string>Libraries</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="libraries_list_widget">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="add_library_btn">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Add Library</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="divider">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="settings_btn">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:#f48c06;
</string>
        </property>
        <property name="text">
         <string>Settings</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="right_section">
     <item>
      <widget class="QWidget" name="library_upper_wrapper" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget{
background-color:#33373d;
}

QLabel {
color:white;
}

QPushButton {
color:#f48c06;
}</string>
       </property>
       <layout class="QHBoxLayout" name="library_upper_layout" stretch="0,0">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QLabel" name="current_library_label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial Black</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Library Name</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="current_library_buttons_layout" stretch="0,0,0">
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="scan_library_btn">
            <property name="maximumSize">
             <size>
              <width>125</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Scan Library</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="update_library_btn">
            <property name="maximumSize">
             <size>
              <width>125</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Update Library</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="new_volumes_btn">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>125</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>New Volumes</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QScrollArea" name="series_scroll_area">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="series_grid_wrapper">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>998</width>
          <height>742</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="series_wrapper_layout_useless">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QGridLayout" name="series_grid_layout"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
